XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -105,9 +116,10 @@ public override void PageLoaded(PageLoadedDirection direction)
                         if (!xenConnection.IsConnected)
                             continue;
 
-                        var host = Helpers.GetMaster(xenConnection);
-                        int index = dataGridViewHosts.Rows.Add(new PatchingHostsDataGridViewRow(host, false));
-                        EnabledRow(host, SelectedUpdateType, index);
+                        var pool = Helpers.GetPoolOfOne(xenConnection);
+                        Host master = pool.Connection.Resolve(pool.master);
+                        int index = dataGridViewHosts.Rows.Add(new PatchingHostsDataGridViewRow(pool, true));

```

Comment:
We need to use a different constructor for the standalone hosts, so that the grid displays the correct icon.